### PR TITLE
test: add missing tests for '_words' helper

### PR DESCRIPTION
**Pull request by abdullahtariq1171**
Added missing tests for `_words`

### CODE REVIEW

File: lib/_internal/_words/_words.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import _words from '.'
+
+it('Returns words in a string', () => {
```
Comment (Beraliv) [inline lib/_internal/_words/_words.test.js]: Start from lower case
Below as well

File: lib/_internal/_words/_words.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  const phrase = 'You only lose when you give up.'
+  const words = _words(phrase)
+
+  expect(words.length).toEqual(8)
```
Comment (Beraliv) [inline lib/_internal/_words/_words.test.js]: Am I missing something or why does it return 8 instead of 7 ?

**Comment by Beraliv** [comment]
@abdullahtariq1171 are you planning to finish PR?

File: lib/_internal/_words/_words.test.js (hunk 2 above)
Comment (abdullahtariq1171) [reply lib/_internal/_words/_words.test.js]: It looks like the issue of regex . See `replace(/([A-Z]+)/g, '_$1')`, Currently it also replaces the first letter of string which it shouldn't

So `_words('You only lose when you give up.')` => `[ '', 'You', 'only', 'lose', 'when', 'you', 'give', 'up.' ]`

**Pull request merged**